gtda/homology/grids.py

```diff
@@ -63,21 +74,24 @@ class CubicalPersistence(BaseEstimator, TransformerMixin):
     <http://gudhi.gforge.inria.fr/doc/latest/group__cubical__complex.html>`_.
 
     """
-    _hyperparameters = {'infinity_values_': [numbers.Number],
-                        '_homology_dimensions': [list, [int, (0, np.inf)]],
-                        'coeff': [int, (2, np.inf)]}
+    _hyperparameters = {'_homology_dimensions': [list, [int, (0, np.inf)]],
+                        'coeff': [int, (2, np.inf)],
+                        '_periodic_dimensions': [list, [int, (0, np.inf)]],
```

Comment:
Shouldn't the elements of the list be booleans? (or, at least, values in the range [0,1]?)

Reply:
Yes!